package/patch-xen.sh

```diff
@@ -10,3 +10,14 @@ fi
 
 echo 'Patched qemu to include --disable-sdl option'
 sed -i 's#source/configure --enable-xen --target-list=i386-softmmu#source/configure --enable-xen --target-list=i386-softmmu --disable-sdl#g' tools/Makefile
+
+gcc -fcf-protection=none -Q --help=target >/dev/null 2>/dev/null
```

Comment:
I really think we ought to just compile with gcc-8 instead of doing this fragile work-around.

Reply:
Unfortunately I'm not able to generate a bootable Xen using gcc-8, after such additional changes:
https://github.com/CERT-Polska/drakvuf/commit/1d6f2a0d6f13b164e0a2ce9c1f6b64c71b0895fc

the resulting hypervisor image is crashing right after "loading initial xen ramdisk".

Reply:
Hmm but I've checked that `gcc-7` is a default GCC for Ubuntu 18.04 and it seems to work properly when built under `gcc-7`.

Ok, I will convert this hack to just use `gcc-7` instead as this is much clearer option.